runtime/jcl/common/stdinit.c

```diff
@@ -123,7 +123,13 @@ standardInit(J9JavaVM *vm, char *dllName)
 		}
 	}
 	/* Now create the classPathEntries */
-	if (initializeBootstrapClassPath(vm) && !IS_RESTORE_RUN(vm)) {
+	switch (initializeBootstrapClassPath(vm)) {
+	case 0:
+		break;
+	case 1:
```

Comment:
Can we use macros instead of 0/1 in order to make this more descriptive? I would also add a function description for `initializeBootstrapClassPath` to document the changes.

Reply:
Yes, I'll do that. I was wanting to deal with the magic number returns from `initializeBootstrapClassPath` (including the failure paths), I'll add some macros.